packages/components/Button/src/Button.types.ts

```diff
@@ -112,6 +112,7 @@ export interface IButtonProps extends Omit<IPressableProps, 'onPress'> {
 
 export interface IButtonSlotProps {
   root: React.PropsWithRef<IViewProps>;
+  ripple?: PressableProps;
```

Comment:
Ooh, an optional slot! I wasn't sure we could do that. I've been wondering how to handle platform specific slots...

I'm gonna link this thread again in case it gives insight because we might not need to add a new slot? https://github.com/microsoft/fluentui-react-native/pull/467/files#r496065158

Reply:
Had a look at that PR already! As per the [docs](https://reactnative.dev/docs/touchablenativefeedback), Pressable API should be preferred. Though, this makes android-button compatible with RN 0.63 and higher.